docs/modules/ROOT/pages/proxies.adoc

```diff
@@ -346,7 +346,7 @@ They can be deployed as-is or used as templates for customization.
 
 Some presets include:
 
-* https://github.com/OpenZeppelin/cairo-contracts/blob/ad399728e6fcd5956a4ed347fb5e8ee731d37ec4/src/openzeppelin/token/erc20/presets/ERC20Upgradeable.cairo[ERC20Upgradeable]
+* https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.4.0b/src/openzeppelin/token/erc20/presets/ERC20Upgradeable.cairo[ERC20Upgradeable]
```

Comment:
If the release changes the version, these links need to be updated right after the release?

Reply:
They'll automatically update from the CI running the version script (as long as the version exists in the URL)